### ISSUE TITLE
IDE is reporting a lot of errors while playing with Colorado COVID example

### ISSUE THREAD

**Issue by sylwiabr** [enso-org/enso#6909]
### What type of issue is this?

Transient – Occurring only once

### Is this issue blocking you from using Enso?

- [ ] Yes, I can't use Enso because of this issue.

### Is this a regression?

- [ ] Yes, previous version of Enso did not have this issue.

### What issue are you facing?

I opened an empty project, went back to the dashboard and opened Covid example from templates. The console was flooded with errors -mostly about visualizations, but few of them were concerning. I am attaching a video and screens from the console.

### Expected behaviour

No errors in the console.

### Screenshots or screencasts

![Zrzut ekranu 2023-05-31 o 15 43 16](https://github.com/enso-org/enso/assets/12892578/1fb8eb79-cfb9-4b6e-8155-0d6a633caf3f)
![Zrzut ekranu 2023-05-31 o 15 42 28](https://github.com/enso-org/enso/assets/12892578/496fb2f2-e860-4d5c-bf9b-4fc5035694b8)
![Zrzut ekranu 2023-05-31 o 15 42 07](https://github.com/enso-org/enso/assets/12892578/4b163bd2-620d-4323-838b-3a6789929838)

video:
https://drive.google.com/file/d/1JATn8h3Vjz4i9qE3IMeUSSV-GMO3s_yW/view?usp=share_link

### Enso Version

Nightly 31.05

### Browser or standalone distribution

Standalone distribution (local project)

### Browser Version or standalone distribution

standalone

### Operating System

MacOS

**Comment by hubertp** [comment]
Partially a duplicate of https://github.com/enso-org/enso/issues/6871

**Comment by sylwiabr** [comment]
I think this one has extra error "already borrowed: BorrowMutError"

**Comment by farmaazon** [comment]
The "already borrowed: BorrowMutError" are actually an echo of the first panic (the panic does not leave borrows, and next frames work on borrowed data).

The question is if the "already borrowed" panics are indeed after the "unwrap at None" panic. If yes, then this is a full duplicate.

**Comment by sylwiabr** [comment]
@farmaazon here are two errors before "BorrowMutError" 
![image](https://github.com/enso-org/enso/assets/12892578/83ed23a9-7f83-40c6-b23e-29eeb43322c9)
the panic was earlier definitely

**Comment by vitvakatu** [comment]
Can't reproduce locally with IDE commit `d44b1250b705f846846a476636b16ffaf308fb52` and engine `5.31`. @sylwiabr could you confirm if the issue still happens for you?

**Comment by sylwiabr** [comment]
can't reproduce on new package
